.github/workflows/ibc-test.yaml

```diff
@@ -0,0 +1,100 @@
+name: IBC test
+on:
+  pull_request:
+    paths:
+      - .github/workflows/ibc-test.yaml
+      - Cargo.toml
+      - Cargo.lock
+      - ci/**
+      - crates/**
+      - tools/**
+  push:
+    branches: main
+      - .github/workflows/ibc-test.yaml
+      - Cargo.toml
+      - Cargo.lock
+      - ci/**
+      - crates/**
+      - tools/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ibc-test:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    env:
+      SRC_DIR: ${{ github.workspace }}/ibc-test-src
+    strategy:
+      fail-fast: false
+      matrix:
+        chain_a:
+          - command: ckb
+            account_prefix: ckb
+          - command: axon
+            account_prefix: axon
+        chain_b:
+          - command: ckb
+            account_prefix: ckb
+          - command: axon
+            account_prefix: axon
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: Create source folder
+        run: mkdir -p $SRC_DIR
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ".\n${{env.SRC_DIR}}/axon"
+      - name: Set Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Install yarn
+        run: npm install --global yarn
+      - name: Prepare CKB
+        run: |
+          if [ ! -f "/tmp/ckb.tar.gz" ]; then
+            curl -L https://github.com/nervosnetwork/ckb/releases/download/v0.107.0/ckb_v0.107.0_x86_64-unknown-linux-gnu.tar.gz -o /tmp/ckb.tar.gz
```

Comment:
why use ckb107 version instead of latest

Reply:
The old testing code is using 107. I do not want to upgrade it in this PR